internal/oidc/provider/dynamic_upstream_idp_provider.go

```diff
@@ -98,10 +98,11 @@ type UpstreamLDAPIdentityProviderI interface {
 }
 
 type StoredRefreshAttributes struct {
-	Username string
-	Subject  string
-	DN       string
-	AuthTime time.Time
+	Username             string
+	Subject              string
+	DN                   string
+	AuthTime             time.Time
+	AdditionalAttributes map[string][]string
```

Comment:
Is `map[string]string` sufficient?  Do we actually intend to keep more than one value?